BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -6,6 +6,7 @@
 //
 
 #import "BOXOAuth2Session.h"
```

Comment:
Do we need to import these two here, or is it sufficient to import 'BOXAbstractSession'? Or even just forward declare it as a class